### PR TITLE
[17.0][MIG] product_supplierinfo_for_customer_sale: Migration to 17.0

### PR DESCRIPTION
**Pull request by xmglord**

### CODE REVIEW

**Comment by xmglord** [comment]
This is waiting for [this](https://github.com/OCA/product-attribute/pull/1617) to be merged.

cc: @luisg123v @desdelinux

**Comment by xmglord** [comment]
@pedrobaeza this is ready for merge.

**Comment by pedrobaeza** [comment]
/ocabot migration product_supplierinfo_for_customer_sale

Reviews are missing

**Comment by xmglord** [comment]
> /ocabot migration product_supplierinfo_for_customer_sale
> 
> Reviews are missing

MB you're right. @luisg123v please review.

**Review by desdelinux — APPROVED** [review]

File: product_supplierinfo_for_customer_sale/README.rst [hunk 1]
```diff
@@ -17,25 +17,26 @@ Product Supplierinfo for Customer Sale
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
-    :target: https://github.com/OCA/sale-workflow/tree/16.0/product_supplierinfo_for_customer_sale
+    :target: https://github.com/OCA/sale-workflow/tree/17.0/product_supplierinfo_for_customer_sale
```
Comment (luisg123v) [inline product_supplierinfo_for_customer_sale/README.rst:20]: Commit title:

execution → autofixes

File: product_supplierinfo_for_customer_sale/tests/test_product_name_search.py [hunk 2]
```diff
@@ -11,7 +11,7 @@ class TestProductNameSearch(TransactionCase):
     """
 
     def setUp(self):
-        super(TestProductNameSearch, self).setUp()
+        super().setUp()
```
Comment (luisg123v) [inline product_supplierinfo_for_customer_sale/tests/test_product_name_search.py]: Isn't this part of the precommit changes? Did you perform this change manually?

File: product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py [hunk 3]
```diff
@@ -0,0 +1,163 @@
+# Copyright 2017 ForgeFlow S.L.
+#   (http://www.forgeflow.com)
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo.tests import Form
+from odoo.tests.common import TransactionCase
+
+
+class TestProductSupplierinfoForCustomerSale(TransactionCase):
+    def setUp(self):
```
Comment (luisg123v) [inline product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py]: `setUpClass`

Same in the other test file.

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py (hunk 3 above)
Comment (xmglord) [reply product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py]: Done

File: product_supplierinfo_for_customer_sale/tests/test_product_name_search.py (hunk 2 above)
Comment (xmglord) [reply product_supplierinfo_for_customer_sale/tests/test_product_name_search.py]: Nop I had to apply it by myself, it is indeed a lint but the autofix didn't change it.

File: product_supplierinfo_for_customer_sale/README.rst (hunk 1 above)
Comment (xmglord) [reply product_supplierinfo_for_customer_sale/README.rst:20]: done

**Review by luisg123v — APPROVED** [review]
LGTM :+1: 

Hi @moylop260 or @pedrobaeza,

Could you review/merge, please?

CC @desdelinux

**Comment by luisg123v** [comment]
@moylop260 or @pedrobaeza,

Friendly reminder, this is approved and waiting for merge.

**Pull request merged**